src/boot/bootstrapper-router.tsx

```diff
@@ -38,42 +32,42 @@ const ContextBridge: FC = () => {
 	return null;
 };
 
-const StandaloneListener: FC = () => {
+const StandaloneListener = (): null => {
 	const { route } = useParams<{ route?: string }>();
 	useEffect(() => {
 		if (route) useAppStore.setState({ standalone: route });
 	}, [route]);
 	return null;
 };
 
-const DefaultViewsRegister: FC<{ t: TFunction }> = ({ t }) => {
+type DefaultViewsRegisterProps = { t: TFunction };
+
+const DefaultViewsRegister = ({ t }: DefaultViewsRegisterProps): null => {
```

Comment:
Probably `t` can be included directly inside this component instead of passing it as prop?